src/ui/api/chart_repository.go

```diff
@@ -164,6 +175,41 @@ func (cra *ChartRepositoryAPI) UploadChartProvFile() {
 	chartController.GetManipulationHandler().UploadProvenanceFile(cra.Ctx.ResponseWriter, cra.Ctx.Request)
 }
 
+//Rewrite the incoming URL with the right backend URL pattern
```

Comment:
UT?

Reply:
UT is covered in Chart controller layer, will not provide UT cases in this layer